lib/zipint.h

```diff
@@ -487,9 +487,12 @@ typedef struct _zip_pkware_keys zip_pkware_keys_t;
 #endif
 #endif
 
+#define _zip_realloc(inout_memory, inout_alloced, element_size, additional_elements) (_zip_realloc)((void **)inout_memory, inout_alloced, element_size, additional_elements)
+int (_zip_realloc)(void **inout_memory, zip_uint64_t *inout_alloced, zip_uint64_t element_size, zip_uint64_t additional_elements);
```

Comment:
Please use a different name for macro and function (like `zip_realloc_implementation` for the function).

Also, since some C compilers warn about symbols beginning with `_` as being reserved, please don't use that prefix for new internal symbols.

Reply:
Done.